adobe-photoshop/adobe-photoshop-quiz.md

```diff
@@ -552,6 +552,7 @@
 
 -[x] anti-aliasing
 
+<<<<<<< HEAD
```

Comment:
```suggestion
<<<<<<< HEAD
```

i think you have to double check there were merge conflict

Reply:
I don't know why I forgot this